### PR TITLE
single controller and readme

### PR DESCRIPTION
**Pull request by vicfriedman**
@jmburges removed multiple controllers + solution

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -11,5 +11,47 @@ class ApplicationController < Sinatra::Base
...
+  end
+
+  get '/sessions/login' do
+
+    # the line of code below render the view page in app/views/sessions/login.erb
+    erb :'sessions/login'
+  end
+
+  post '/sessions' do
+    # find the user who submitted the log in forms by looking in your database
```
Comment (jmburges) [inline app/controllers/application_controller.rb]: I want to discourage students from thinking of comments as a place to put paragraphs of text. Can you remove thes?

File: spec/controllers/application_controller_spec.rb [hunk 2]
```diff
@@ -16,4 +16,66 @@
...
+    end
+
+    it "renders the signup template" do
+      expect(last_response.body).to include("Log In")
+    end
+  end
+
+  describe "user's home page: GET /users/home" do
+    
+    # before(:each) do
```
Comment (jmburges) [inline spec/controllers/application_controller_spec.rb]: Commented out code?

**Comment by jmburges** [comment]
few small issues and :shipit:

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (vicfriedman) [inline app/controllers/application_controller.rb]: done

File: spec/controllers/application_controller_spec.rb (hunk 2 above)
Comment (vicfriedman) [inline spec/controllers/application_controller_spec.rb]: done

**Pull request merged**